@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -1042,7 +1042,15 @@ export class TileGeometryCreator {
                     MapMaterialAdapter.create(edgeMaterial, {
                         color: buildingTechnique.lineColor,
                         objectColor: buildingTechnique.color,
-                        opacity: buildingTechnique.opacity
+                        opacity: buildingTechnique.opacity,
+                        lineWidth: (frameMapView: MapAdapterUpdateEnv) => {
+                            // lineWidth for ExtrudedPolygonEdges only supports 0 or 1
+                            const value = getPropertyValue(
+                                buildingTechnique.lineWidth,
+                                frameMapView.env
+                            );
+                            return Math.max(0, Math.min(1, Math.round(value)));
+                        }
```

Comment:
maybe it would be safer to write something like
```
return typeof value === "number" && value > 0 ? 1 : 0;
```

Reply:
done